pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -312,50 +392,18 @@ func (e *DockercfgController) createTokenSecret(serviceAccount *api.ServiceAccou
 			Annotations: map[string]string{
 				api.ServiceAccountNameKey: serviceAccount.Name,
 				api.ServiceAccountUIDKey:  string(serviceAccount.UID),
+				PendingDockercfgNameKey:   "true",
```

Comment:
> why add a pending annotation we have to remove? why not just annotate this to indicate it was created by / owned by this controller, and we can leave the annotation in place? we can tell if a token is pending by the lack of a token in the data map

A controllerRef, how difficult you are.  It's owned by this controller, not the controller that fills in the secret?  It's owned by both?  It's an ownerRef pointing to an object that doesn't exist?